Demo/Demo.csproj

```diff
@@ -13,8 +13,8 @@
     <ProjectReference Include="..\Src\Fido2\Fido2.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
-    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.0" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
```

Comment:
Is this condition necessary? It always seems to be evaluated to true.

Reply:
removed.